tools/symbol-extractor/symbol_extractor.ts

```diff
@@ -74,14 +74,14 @@ export class SymbolExtractor {
 
   static diff(actual: Symbol[], expected: string|((Symbol | string)[])): {[name: string]: number} {
     if (typeof expected == 'string') {
-      expected = JSON.parse(expected);
+      expected = JSON.parse(expected) as (Symbol | string)[];
```

Comment:
I don't think that JSON.parse can produce `Symbol` objects so...

```suggestion
      expected = JSON.parse(expected) as string[];
```